### PR TITLE
Add content for practical Object-Oriented Python session

### PR DESCRIPTION
**Pull request by eelanagaraj**
Hope the last activity isn't way too simple or complex -- the goal here is to have some flexibility so groups can tackle a variety of stuff with little pressure to get through everything. Let me know if there are other exercises that you think are a better fit or if these ones are too hard! :)

### CODE REVIEW

File: content/docs/python/python_classes_practical.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## 2. Kahoot
+
+Let's do [a Kahoot](https://create.kahoot.it/details/1b6aba45-138f-4af3-bb3a-3c8e290a26d5) to check our understanding of the concepts covered last class!
+
+## 3. Exercises (part 1)
+
+In groups, go through the exercises on [this tutorial](https://pynative.com/python-object-oriented-programming-oop-exercise/). Try not to look at the solutions until you have finished the problem. Copy the skeleton code into your local development setup (VSCode, PyCharm, or Google Colab) and try to achieve the "expected output" described for each exercise. Make sure everyone in your group has understood and produced the proper output before moving on to the next task in this section :)
+
+## 4. Exercises (part 2)
```
Comment (emilywoods) [inline content/docs/python/python_classes_practical.md:43]: I think this section could be challenging, so perhaps after the class we could share solutions with them?

File: content/docs/python/python_classes_practical.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+| Exercises in groups                                    | 30 minutes           | ❌                 |
+| Break                                                  | 10 minutes           | ❌                 |
+| Exercises in groups                                    | 40 minutes           | ❌                 |
+| Check-out                                              | 10 minutes           | ❌                 |
+
+## 0. Lesson Goals
+
+- Practice object-oriented programming concepts
+
+## 1. Check-In
```
Comment (emilywoods) [inline content/docs/python/python_classes_practical.md:31]: good idea!

**Review by emilywoods — APPROVED** [review]
looks good to me!

I think the last question will be a challenge because it requires some thinking about what they've learned from so far and applying it for themselves. But we will be there to guide them if so :D

File: content/docs/python/python_classes_practical.md (hunk 1 above)
Comment (eelanagaraj) [reply content/docs/python/python_classes_practical.md:43]: @emilywoods  totally! I also always get nervous that they will run out of exercises lol but I think I will also change the "optional part 4" to more explicitly be "final project ideas" + extensions. They are probably all too hard for the scope of this class but maybe would be interesting for some of them to think about. What do you think?

**Pull request merged**